generic/tests/cfg/netperf.cfg

```diff
@@ -37,6 +37,8 @@
     sizes_rr = "64 256 4096"
     #client configuration
     client = localhost
+    # please fix physical nic name of client for jumbo frame case
+    client_physical_nic =
```

Comment:
Is this `client_physical_nic` need to be manually assigned every time?
If it is yes, I would suggest you comment this param out, or test if this param is not provided, skip the test. 
maybe like this:
`# please config physical nic name of client for jumbo frame case by uncommenting it `
`# or the test will be skipped`
`# client_physical_nic = <your_client_physical_nic>`